app/models/link.rb

```diff
@@ -18,10 +18,8 @@ class Link < ApplicationRecord
       .where(service_interactions: { service_id: service })
   }
 
-  HTTP_OK_STATUS_CODE = 200
-
-  scope :good_links, -> { where(status: HTTP_OK_STATUS_CODE) }
-  scope :currently_broken, -> { where.not(status: HTTP_OK_STATUS_CODE) }
+  scope :good_links, -> { where(status: "ok") }
+  scope :currently_broken, -> { where.not(status: "ok") }
```

Comment:
I'd think we only consider links with a status of broken as `currently_broken` Would have thought caution would be a separate grouping. Or if it'll be just one grouping it could be "For Review" rather than "Broken"? What do you think? Maybe Mia will have thoughts?